src/main/java/org/openstreetmap/atlas/checks/maproulette/data/ProjectConfiguration.java

```diff
@@ -0,0 +1,70 @@
+package org.openstreetmap.atlas.checks.maproulette.data;
+
+/**
+ * Helper class to decouple MapRouletteConfiguration from configuring projects. This allows easier
+ * updating of code that handles projects without touching general Map Roulette configuration code.
+ *
+ * @author nachtm
+ */
+public class ProjectConfiguration
+{
+    private String name;
+    private String description;
+    private String displayName;
+
+    /**
+     * Defines a basic project, where all optional fields default to name.
+     * 
+     * @param name
+     *            The name of the project
+     */
+    public ProjectConfiguration(final String name)
+    {
+        this.name = name;
+        this.description = name;
+        this.displayName = name;
+    }
+
+    /**
+     * Defines a project and all of its fields.
+     * 
+     * @param name
+     *            The name of the project
+     * @param description
+     *            The description of the project
+     * @param displayName
+     *            The name displayed on Map Roulette for the project
+     */
+    public ProjectConfiguration(final String name, final String description,
+            final String displayName)
+    {
+        this.name = name;
```

Comment:
While you're here - are there other project specific fields we can add ?

Reply:
I just added a boolean `enabled` since that was a pretty straightforward add. There's also an object `groups` that the endpoint takes, but I don't know much about what it's used for or how it's used. If you'd like me to dig into it I can but it should also be easy to add further down the line if we discover that we need it

Reply:
I think this is good - thanks !